crates/uv/tests/tool_run.rs

```diff
@@ -776,13 +776,11 @@ fn tool_run_list_installed() {
     success: false
     exit_code: 2
     ----- stdout -----
-    Provide a command to invoke with `uv tool run <command>` or `uv tool run --from <package> <command>`.
+    Provide a command to run with `uv tool run <command>`.
 
-    The following tools are already installed:
+    The following tools are installed:
 
-    black v24.2.0
-    - black
-    - blackd
+    - black v24.2.0
```

Comment:
I went back and forth on a newline here. I _think_ it looks better if we have an extra newline separating the list? Relevant for #7687

Reply:
The extra line looks way cleaner to me. If you're okay with me putting it here, I will change the other PR.

Reply:
👍 yeah sorry I think I suggested you remove it over there